### DISCUSSION TITLE
Is it possible to disable code autoformatting on alt + tab / window focus lost?

### DISCUSSION THREAD

**Discussion by nerfi58** [redhat-developer/vscode-java#2881] [Q&A] [answered]
Everytime I use alt + tab or manualy change focus to another window with my mouse code gets formatted. Is there any option to disable it? I tried with `"java.format.onType.enabled": false`, but it didn't work.

**Comment by fbricon** [comment, marked as answer]
Did you set those options?
```
  "editor.formatOnSave": true,
  "files.autoSave": "onFocusChange",
```

**Reply by nerfi58** [reply]
Oh idk why, but I had `"files.autoSave": "onFocusChange"` in my workspace, even though I don't remember setting it. Thanks.
